### DISCUSSION TITLE
BaseUser without email

### DISCUSSION THREAD

**Discussion by jeanlst** [fastapi-users/fastapi-users#858] [Q&A]
Hey,

Do I have to use by default the Base classes provided by fastapi-users? The field email for example is of no interest for me, will everything work if I just create a new User inheriting from CreateUpdateDictModel and all the fields except email? What's the best way to go on about that?

**Comment by frankie567** [comment]
Hi @jeanlst,

Unfortunately, the email is quite core everywhere in the implementation, so I fear it would be a bit hard to avoid it.

**Reply by jeanlst** [reply]
Well, that's unfortunate since I'm not to use e-mail at all for ungodly reasons in the api...
